src/shell/store/content.js

```diff
@@ -406,7 +406,15 @@ export function saveItem(itemZUID, action = "") {
     if (item.web.metaDescription) {
       item.web.metaDescription = item.web.metaDescription.slice(0, 160);
     }
-
+    if (item.web.metaTitle) {
+      item.web.metaTitle = item.web.metaTitle.slice(0, 150);
+    }
+    if (item.web.metaLinkText) {
+      item.web.metaLinkText = item.web.metaLinkText.slice(0, 150);
+    }
+    if (item.web.metaKeywords) {
+      item.web.metaKeywords = item.web.metaKeywords.slice(0, 250);
```

Comment:
Why is 250 here? The ui character counter says it's 255.
![image](https://github.com/zesty-io/manager-ui/assets/28705606/60c4254e-b047-41bc-96f1-be076d3343eb)

Reply:
done updating changes

Reply:
@glespinosa which character count did we settle on and why?

Reply:
@shrunyan  I'm basing on the UI
![image](https://github.com/zesty-io/manager-ui/assets/44116036/c8f51167-beb1-440b-9479-4357648a9caa)